server/src/test/java/org/elasticsearch/cluster/metadata/IndexNameExpressionResolverTests.java

```diff
@@ -1702,26 +1717,26 @@ public void testIndicesAliasesRequestTargetDataStreams() {
         {
             IndicesAliasesRequest.AliasActions aliasActions = IndicesAliasesRequest.AliasActions.add().index(dataStreamName);
             IllegalArgumentException iae = expectThrows(IllegalArgumentException.class,
-                () -> indexNameExpressionResolver.concreteIndexNames(state, aliasActions, false));
+                () -> indexNameExpressionResolver.concreteIndexNames(state, aliasActions));
             assertEquals("The provided expression [" + dataStreamName + "] matches a data stream, specify the corresponding " +
                 "concrete indices instead.", iae.getMessage());
         }
 
         {
             IndicesAliasesRequest.AliasActions aliasActions = IndicesAliasesRequest.AliasActions.add().index("my-data-*").alias("my-data");
             IllegalArgumentException iae = expectThrows(IllegalArgumentException.class,
-                () -> indexNameExpressionResolver.concreteIndexNames(state, aliasActions, false));
+                () -> indexNameExpressionResolver.concreteIndexNames(state, aliasActions));
             assertEquals("The provided expression [my-data-*] matches a data stream, specify the corresponding concrete indices instead.",
                 iae.getMessage());
         }
 
-        {
-            IndicesAliasesRequest.AliasActions aliasActions = IndicesAliasesRequest.AliasActions.add().index(dataStreamName)
-                .alias("my-data");
-            String[] indices = indexNameExpressionResolver.concreteIndexNames(state, aliasActions, true);
-            assertEquals(1, indices.length);
-            assertEquals(backingIndex.getIndex().getName(), indices[0]);
-        }
+        //{
```

Comment:
why is this commented out?

Reply:
I missed that in an earlier commit. The backport of https://github.com/elastic/elasticsearch/pull/57900/ will un-comment it.